EIPS/eip-9999.md

```diff
@@ -0,0 +1,67 @@
+---
+eip: 9999
+title: Generalized consolidation requests.
+description: Reutilizes the consolidation request contract to pass general messages from the EL to the CL.
+author: Potuz (@potuz)
+discussions-to: https://ethereum-magicians.org/t/eip-9999-generalized-consolidation-requests/11111
```

Comment:
Please create a discussions topic in Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5